examples/mongodbatlas_stream_processor/main.tf

```diff
@@ -0,0 +1,43 @@
+resource "mongodbatlas_stream_instance" "example" {
+  project_id    = var.project_id
+  instance_name = "InstanceName"
```

Comment:
Maybe use a variable for instance_name?

Reply:
IMO it does not make a difference to use a variable rather than a string directly for the example.

Reply:
A bit nit, feel free to resolve